spec/config/key_manager_spec.rb

```diff
@@ -18,6 +18,23 @@
       expect(config.project_config).to eq(project_config)
       expect(config.repo_config).to eq(repo_config)
     end
+
+    it "raises an error if config can't be found" do
+      expect { KeyManager.config_from_project('justkidding') }.to(
+        raise_error(Txgh::ProjectConfigNotFoundError)
+      )
+    end
+
+    it "raises an error if the scheme can't be recognized" do
+      # remove the scheme
+      allow(Txgh::Config::KeyManager).to(
+        receive(:raw_config).and_return(YAML.dump(base_config))
+      )
+
+      expect { KeyManager.config_from_project(project_name) }.to(
+        raise_error(InvalidProviderError)
```

Comment:
Oh actually... the tests usually include `Txgh` at the top. But I can make this more consistent.